include docker image example

sando38 opened the pull request:
Hello Holger,

I updated my eturnal docker image and thought, this may be also of interest for your repository... and potential other users to try eturnal.

It is basically eturnal packaged as a docker image, without any customization.

There is a readme included, also available on [DockerHub](https://hub.docker.com/r/sando38/docker-eturnal), which shows usage, etc. Of course, it should be altered to fit your corporate identity, if you decided to merge this with your repository.

I am continue to maintain the image, however, haven't setup a github repo, yet.

It is scanned by trivy with no vulnerabilities:
```
2022-05-17T15:41:57.871+0100    INFO    Detected OS: alpine
2022-05-17T15:41:57.871+0100    INFO    Detecting Alpine vulnerabilities...
2022-05-17T15:41:57.874+0100    INFO    Number of language-specific files: 0

sando38/docker-eturnal:latest (alpine 3.15.4)
=============================================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```

What I have not tested is using environment variables, I mount a custom `eturnal.yml`. It should work however as if one uses eturnal as standalone application.

I could also offer to maintain the image in this repo with a github action linked to it.

Best regards and have a great day,
sando

weiss commented:
Awesome, thanks a lot!  I have no time right now but will have a look next week.

I'm not really into Docker, so if you're motivated to maintain this, that would be cool.

(Could you maybe [drop me an email](mailto:holger@zedat.fu-berlin.de), or if you happen to be using XMPP, a message to `holger@jabber.fu-berlin.de`, just so that I have some contact info?)

sando38 commented:
Thanks. Take you time, I am busy until next week as well. 

Going to contact you via xmpp later!

Have a good evening.

sando38 commented:
Hi Holger,

I updated the Dockerfile to simplify the erlang-cookie generation as discussed. I adjusted the README accordingly.

Furthermore I made some cleanups of the Dockerfile.
